compiler/semantic/op.go

```diff
@@ -931,9 +928,9 @@ func semCallOp(scope *Scope, call *ast.Call, seq dag.Seq) (dag.Seq, error) {
 	if !function.HasBoolResult(call.Name) {
 		return nil, nil
 	}
-	c, err := semCall(scope, call)
+	out, err := a.semCall(call)
```

Comment:
Nit: Leave this name unchanged.
```suggestion
	c, err := a.semCall(call)
```